plugins/database/mongodb/connection_producer.go

```diff
@@ -29,6 +31,9 @@ type mongoDBConnectionProducer struct {
 	WriteConcern  string `json:"write_concern" structs:"write_concern" mapstructure:"write_concern"`
 	Username      string `json:"username" structs:"username" mapstructure:"username"`
 	Password      string `json:"password" structs:"password" mapstructure:"password"`
+	SSLCert      string `json:"ssl_cert" structs:"ssl_cert" mapstructure:"ssl_cert"`
```

Comment:
These should all be labeled TLS instead of SSL.

Reply:
Good catch